### DISCUSSION TITLE
RemoveEventListener error in react-native-navigation@4

### DISCUSSION THREAD

**Discussion by gkasireddy202** [react-navigation/react-navigation#11933] [Q&A]
I have used react-navigation: 4.4.4 in my project for the last 5 years.
I upgraded the react-navigation to 0.73.2 and got this error.
_reactNative.Linking.removeEventListener is not a function (it is undefined)

![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-13 at 12 28 02](https://github.com/react-navigation/react-navigation/assets/90603484/1847200d-16a9-4cf9-8322-7a8c010f7704)

**Comment by gkasireddy202** [comment]
![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-13 at 12 28 02](https://github.com/react-navigation/react-navigation/assets/90603484/98f3fe40-9584-4614-b884-2df329615618)

**Comment by radko93** [comment]
This React Navigation 4 version is not maintained anymore so it won't be updates. Only latest React Navigation (6) is supported currently and it should be used with new React Native versions.

**Comment by gkasireddy202** [comment]
@radko93 - But in navigation documentation.Says like below.
I think this issue cause because of listeners.
<img width="981" alt="Screenshot 2024-04-15 at 4 16 26 PM" src="https://github.com/react-navigation/react-navigation/assets/90603484/7ca06cb3-591f-4997-a632-7b361b131506">

**Reply by radko93** [reply]
This is an old version of the docs you can see that on the top of the article.

**Reply by gkasireddy202** [reply]
Yes.But how to resolve this error?

**Reply by radko93** [reply]
It might need an update to React Navigation. I have no idea.

**Reply by gkasireddy202** [reply]
My project has more screens with having react-native-navigation@4 navigation structure. It is difficult to upgrade to the latest version.
